Lacework/Applications_and_Dashboards/Lacework_Alerts_Overview.json

```diff
@@ -0,0 +1,235 @@
+{
+    "type": "DashboardV2SyncDefinition",
+    "name": "Lacework Alerts Overview",
+    "description": "",
+    "title": "Lacework Alerts Overview",
+    "rootPanel": null,
+    "theme": "Light",
+    "topologyLabelMap": {
+        "data": {}
+    },
+    "refreshInterval": 0,
+    "timeRange": {
+        "type": "BeginBoundedTimeRange",
+        "from": {
+            "type": "RelativeTimeRangeBoundary",
+            "relativeTime": "-1d"
+        },
+        "to": null
+    },
+    "layout": {
+        "layoutType": "Grid",
+        "layoutStructures": [
+            {
+                "key": "panel720080EC8414794E",
+                "structure": "{\"height\":8,\"width\":11,\"x\":0,\"y\":0}"
+            },
+            {
+                "key": "panel19E38DB09E01884E",
+                "structure": "{\"height\":8,\"width\":8,\"x\":8,\"y\":8}"
+            },
+            {
+                "key": "panel2642112C9765794A",
+                "structure": "{\"height\":8,\"width\":8,\"x\":0,\"y\":8}"
+            },
+            {
+                "key": "panelC3D24447BFF9EB41",
+                "structure": "{\"height\":8,\"width\":13,\"x\":11,\"y\":0}"
+            },
+            {
+                "key": "panelD8288D4D987ED945",
+                "structure": "{\"height\":8,\"width\":8,\"x\":16,\"y\":8}"
+            },
+            {
+                "key": "panel9CAA8CC8B00E4B4A",
+                "structure": "{\"height\":13,\"width\":24,\"x\":0,\"y\":16}"
+            }
+        ]
+    },
+    "panels": [
+        {
+            "id": "D11FFF277C809767",
+            "key": "panel720080EC8414794E",
+            "title": "Count of Events",
+            "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"table\"},\"series\":{}}",
+            "keepVisualSettingsConsistentWithParent": true,
+            "panelType": "SumoSearchPanel",
+            "queries": [
+                {
+                    "queryString": "_sourceCategory=*lacework*\n| json field=_raw \"event_title\"\n| json field=_raw \"event_description\"\n| json field=_raw \"event_severity\" as sev\n| json field=_raw \"event_type\"\n| json field=_raw \"event_source\"\n| where sev matches \"{{sev}}\"\n| count by event_title, sev\n| sort by _count",
```

Comment:
_sourceCategory=*lacework*

I'm assuming that the `*lacework*` is the part that might change between users?

so that users in the future know to make it:
"$$lacework"
Then add a note in the readme )look at others for examples of this) telling the use to find/replace `$$lacework` for whatever sourcecategory they are going to use

example

"Replace $$lacework for yout search pattern for laceworks data, ie:
'_sourceCategory=*lacework*`, etc."